odpi.bom

```diff
@@ -132,7 +132,16 @@ bigtop {
                 ref = 'branch-2.7-ODP'; dir = 'branch-2.7-ODP'}
 ***/
     }
-
+    'hive' {
+      name    = 'hive'
+      relNotes = 'Apache Hive'
+      version { base = '1.2.1'; pkg = base+'+odpi'; release = 1 }
+      tarball { destination = "apache-${name}-${version.base}-src.tar.gz"
+                source      = destination }
+      url     { download_path = "/$name/$name-${version.base}/"
+                site = "${apache.APACHE_MIRROR}/${download_path}"
```

Comment:
I actually think we need to include mirror of the Hive repo under odpi GH complete with ODPi specific branch (same deal as Hadoop or Bigtop) and use that URL here. E.g. https://github.com/odpi/hadoop/commits/branch-2.7-ODP